app/views/tasks/new.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1>Task#new</h1>
+
+<%= simple_form_for @task, url: tasks_path, method: :post do |f| %>
+<%= f.input :name %>
+<%= f.input :description %>
+<%= f.input :due_date %>
+<%= f.button :submit, 'Create task' %>
```

Comment:
I think these `inputs` and `button` should be nested 1 level deeper 😅

Reply:
Oops 🙊 will fix it